### DISCUSSION TITLE
Starting Details for Trust System Team

### DISCUSSION THREAD

**Discussion by evnd04** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#9] [Informational]
Trust System Team responsibilities:

> Managing rumors, sighted reports, and trust levels. Trust levels will be calculated and tracked.
> Interfaces for reporting rumors and sighted reports.
> Storing and retrieving trust-related data
> For simplicity’s sake and efficiency in finding the category of the report, the UI should be a screen with “Buttons” that have the type of report to make. e.g., safety issue, suspicious person, armed suspect, sexual harasser, etc)
> Alongside the above buttons, a button to immediately send out specific alerts (SOS, Panic Alert, Immediate help, etc) This goes hand in hand with the Admin Interface Team.
> Report System template:
> Title: [Text Input] - Short, descriptive title of the report.
> Description: [Text Area] - Detailed description of the issue or incident.
> Location: [Map with Pin Drop/Address Input] - Option to specify the location using an interactive map or by entering an address. Integrate with mapping libraries like Google Maps or Mapbox. This goes hand in hand with the location management team.
> Submit Button: [Button] - To submit the report. Provide a confirmation message upon successful submission.

provide name, email, GitHub username to be a part of this team, if interested in being a team leader add (team leader)

Team Leader: @Alejandro-DeJesus 

Developers:
Diego Velez @diego-velez-upr (Dev/Documentation)
Jorge Rivera @JorgeRivera94 (Dev/Documentation)
Natanael Batista Alvarez @NathBat09 (Dev)
Jon Vargas Torres @jonVargas1 (Dev)

This team is currently *OPEN*
Group chat link: https://chat.whatsapp.com/CZrfHcyLmhK3LqJYrzTojb

**Please note:** Two of the members in this team need to take on the responsibility of handling both documentation (while reporting to @lex939 on this) and software development for the trust system. These two members will split their time 50-50 between documentation and development, so each will do a bit less of each task to collectively meet the same total of tasks per milestone. Please come to an agreement as a team on which people will be filling these roles.

**Comment by ghost** [comment]
Diego Vélez
diego-velez-upr
diego.velez6@upr.edu

**Comment by JorgeRivera94** [comment]
Jorge Rivera
jorge.rivera94@upr.edu 
JorgeRivera94

**Comment by Alejandro-DeJesus** [comment]
Alejandro De Jesus
Alejandro-Dejesus
Alejandro.dejesus1@upr.edu
(Team Leader if no one else is interested)

**Comment by NathBat09** [comment]
Natanael Batista Alvarez
natanael.batista@upr.edu
NathBat09

**Comment by jonVargas1** [comment]
Jon Vargas Torres
jon.vargas@upr.edu
jonVargas1
